module/VuFind/src/VuFind/RecordDriver/MarcAdvancedTrait.php

```diff
@@ -220,9 +216,12 @@ public function getBibliographyNotes()
      */
     public function getFilteredXML()
     {
-        $record = clone $this->getMarcRecord();
+        $record = clone $this->getMarcReader();
         // The default implementation does not filter out any fields
+        // $marc = new \File_MARCXML($record->toXML(), \File_MARCXML::SOURCE_STRING);
         // $record->deleteFields('9', true);
```

Comment:
Does this $record need to be changed to $marc for the example to work, or am I misunderstanding what's going on here?

Reply:
Yes, the example was borked.